docs/starters.yml

```
@@ -1666,3 +1666,13 @@
   features:
     - Includes Contentstack Delivery API for any environment
     - Dynamic content from Contentstack CMS
+- url: https://gatsby-craftcms-barebones.netlify.com
+  repo: https://github.com/frankievalentine/gatsby-craftcms-barebones
+  description: Barebones setup for using Craft CMS and Gatsby locally.
+  tags:
+    - Craft
```

It seems that "Craft CMS" is more common looking at their own website, social media profiles etc.

```suggestion
    - Craft CMS
```

---

Thanks @LekoArts!